src/quote/swap/adapters/static/swap-data-config.ts

```diff
@@ -14,164 +39,125 @@ export const SwapDataConfig: Readonly<{
 }> = {
   1: {
     'ETH2X': {
-      '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2': {
+      [mainnet.weth]: {
```

Comment:
nit: next time we have two distinct features like (address handling cleanup) and (uXRP support) lets separate their PRs